modules/nf-core/parabricks/genotypegvcf/main.nf

```diff
@@ -0,0 +1,62 @@
+process PARABRICKS_GENOTYPEGVCF {
+    tag "$meta.id"
+    label 'process_high'
+
+    container "nvcr.io/nvidia/clara/clara-parabricks:4.2.0-1"
+
+    /*
+    NOTE: Parabricks requires the files to be non-symlinked
+    Do not change the stageInMode to soft linked! This is default on Nextflow.
+    If you change this setting be careful.
+    */
+    stageInMode "copy"
+
+    input:
+    tuple val(meta), path(input, stageAs:'')
```

Comment:
is this `stageAs` allowed to have an empty string? I wonder if that's why your tests are failing.

Reply:
I thought it would work because it did in the indexgvcf module for the same purpose. But apparently there is still something wrong because the tests are failing

Reply:
Why was the `stageAs:''` required for the input of indexgvcf and not here?